shenyu-admin/src/main/java/org/apache/shenyu/admin/service/register/AbstractShenyuClientRegisterServiceImpl.java

```diff
@@ -196,7 +200,40 @@ public String doRegisterURI(final String selectorName, final List<URIRegisterDTO
         }
         return ShenyuResultMessage.SUCCESS;
     }
-
+    
+    @Override
+    public String doHeartbeat(final String selectorName, final List<URIRegisterDTO> uriList) {
+        if (CollectionUtils.isEmpty(uriList)) {
+            return "";
```

Comment:
return null?

Reply:
i copied this from 'org.apache.shenyu.admin.service.register.AbstractShenyuClientRegisterServiceImpl#doRegisterURI'